By default, the build system will download an appropriate JDK to build
the project.  Alter the build system to instead search for an installed
JDK.

--- pom.xml.orig	2026-08-18 13:13:06
+++ pom.xml	2026-09-01 16:50:12
@@ -390,17 +390,10 @@
           <executions>
             <execution>
               <goals>
-                <goal>toolchain</goal>
+                <goal>select-jdk-toolchain</goal>
               </goals>
             </execution>
           </executions>
-          <configuration>
-            <toolchains>
-              <jdk>
-                <version>26</version>
-              </jdk>
-            </toolchains>
-          </configuration>
         </plugin>
         <plugin>
           <groupId>org.codehaus.mojo</groupId>
